docs/installation.md

```diff
@@ -58,10 +58,8 @@ This method can be used to run federated learning experiments in an isolated env
 
 * Pull the latest image:
 
-	> **Note:** OpenFL image hosted on `docker.io` has not been updated since the 1.5 release due to a change in namespace. We are working on this issue. In the meantime, use the instructions below to build an image from source.
-
 	```bash
-	docker pull ghcr.io/securefederatedai/openfl/openfl:latest
+	docker pull ghcr.io/securefederatedai/openfl:latest

```

Comment:
could you also update it here?
https://github.com/securefederatedai/openfl/tree/develop/openfl-docker#base-image

Reply:
Migrated `openfl-docker` readme to official docs with your suggested corrections in https://github.com/securefederatedai/openfl/pull/1318/commits/89e3fbb97c948504b4c19cfa420fc1298a6f8ff3

Please note that this is one step of consolidation. I will rewrite the guide in a new PR